apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -205,6 +205,12 @@ const EventTypePage = (props: EventTypeSetupProps) => {
         message = `${err.data.code}: ${t(err.message)}`;
       }
 
+      if (err.data?.code === "INTERNAL_SERVER_ERROR") {
+        const message = t("unexpected_error_try_again");
+        showToast(message, "error");
+        return;
```

Comment:
```suggestion
        message = t("unexpected_error_try_again");
```

Reply:
@leog - thanks for reviewing the code, I have made the above suggested change